### PR TITLE
[Java] Binary Tree Traversals (Inorder, Postorder & Preorder)

### PR DESCRIPTION
**Pull request by Vanshikaa00**
Hello, i added the solution for the dfs traversal of a binary tree in java folder.

### CODE REVIEW

File: Java/Binary Tree DFS Traversal/binaryTreeDFS.java [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import java.util.*;
+
+public class binaryTreeDFS {
```
Comment (amanmehara) [inline Java/Binary Tree DFS Traversal/binaryTreeDFS.java:18]:
```suggestion
public class BinaryTreeDFS {
```

**Review by amanmehara — CHANGES_REQUESTED** [review]

**Comment by Vanshikaa00** [comment]
Made the requested changes @amanmehara

**Review by amanmehara — CHANGES_REQUESTED** [review]
You would also have to change the file name to be same as the class name.

https://docs.oracle.com/javase/tutorial/java/package/managingfiles.html

**Review by amanmehara — APPROVED** [review]
Thanks for contributing! 🎉

**Comment by Vanshikaa00** [comment]
Thank you @amanmehara 😄 !!

**Pull request merged**